github-discussions-bringing-the-open-source-community-closer-together-and-all-in-github.md

```diff
@@ -0,0 +1,235 @@
+
+Currently, the next.js repository README only leads its community members to two places: Discussions and Discord. Discussions is for asking questions, discussing ideas, and sharing new projects, and Discord is for chatting.
+
+![The Community section on the next.js repository homepage](media/the-community-section-on-the-next-js-repository-homepage.png)
+<div class="caption-center"> The Community section on the next.js repository homepage </div>
+
+The above examples are a sign that **the shape of open source communities is changing from scattered to more unified.**
+
+**Having the community running in just one platform not only reduces the cost of participation for community members, but also, to some extent, relieves the stress on project maintainers.**
+
+It seems hopeful to me that the future of open source communities would be:
+
+<span style="text-align:center;text-decoration:underline;font-weight:bold">All in GitHub</span>
+
+* **GitHub Issues** is for collecting bug reports and new feature requests.
+* **GitHub Discussions** is for publishing project information, including at least three types:
+    * RFCs for new features
+    * Learning materials and project announcements
+    * Q&A and other types of communication
+* **GitHub Projects** is for tracing development progress. Community members can participate in the collaboration.
+* **GitHub Pull Request** is for committing code. Community members can take part in the code review.
+
+With Discussions, project information alignment becomes centralized, transparent, and faster.
+
+Every question and answer made by community members is a contribution to the community knowledge base, and every 👍 they left in the RFC discussion page takes its part in the project's decision making.
+
+Every author of an open source project has probably dreamed of building an active, self-governed, and contributing open source community that accelerates good iterations of the project.
+
+It seems that we are now one step closer to that dream.
+
+## Thinking beyond the feature
+
+GitHub is an awesome product. It has even introduced more useful and thoughtful features since its acquisition by Microsoft in 2018.
+
+As a fan of GitHub, I am very glad that this company keeps expanding the possibility of its product and keeps introducing new features that make millions of users' eyes brighten.
+
+On [Hacker News](https://news.ycombinator.com/item?id=22388639), however, some developers expressed their concern about the expansion of the "GitHub Kingdom":
+
+> GitHub is slowly eating the surrounding cities. Github pages replaced our hosted sites. Projects replaced Trello/Waffle and friends. Actions replaced Travis/CircleCI. Sponsors is about to hit Patreon, and now Discussions aims for a piece of StackOverflow pie…
+> I think they are doing great in terms of performance and UI. It's hard to resist the convenience of having everything linked together in a coherent way. But this will probably backfire in the future.
```

Comment:
```suggestion
>
> I think they are doing great in terms of performance and UI. It's hard to resist the convenience of having everything linked together in a coherent way. But this will probably backfire in the future.
```